common/components/common/integrations/NewsletterSignup.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import React from 'react';
+
+class NewsletterSignup extends React.Component {
+
+render() {
+  return (
+    <div id="mc_embed_signup">
+      <form action="https://democracylab.us3.list-manage.com/subscribe/post?u=72af92d0a817dcbf3aa960ee0&amp;id=d3b4c4d81c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" className="validate" target="_blank" noValidate>
```

Comment:
Are there any mailchimp id's in here that would make sense to put in an environment variable?

Reply:
Right now, I don't think so, but possibly in the future. I'm not entirely familiar with what Mailchimp needs to be unique per instance of a signup form, so I'd rather not set variables that may not need to be variables until we have a need.